exercises/practice/triangle/.meta/example.zig

```diff
@@ -1,6 +1,6 @@
 pub const TriangleError = error{
     Degenerate,
-    InvalidInequality,
+    Invalid,
 };
```

Comment:
Do other tracks even make this distinction?

Reply:
Note to myself:

Degenerate = Zero area, the "triangle" is actually a line with 3 points on it

Invalid = Can't be a triangle

(At least, on a flat Euclidean plane)

Reply:
Thanks for the deep dive!